packages/gitbook/src/components/primitives/Button.tsx

```diff
@@ -1,6 +1,7 @@
 'use client';
 
 import type { ButtonHTMLAttributes, HTMLAttributeAnchorTarget, HTMLAttributes } from 'react';
+import { forwardRef } from 'react';
```

Comment:
You have 2 import from `react`, let's do a clean `import * as React from 'react';`